### PR TITLE
Refine MPUpdated contract into several contracts

### PR DESCRIPTION
**Pull request by erduna**
The larger MPUpdated contract has been refined with SME into three smaller more relevant contracts which are explicit to what is changed from a business perspective.

Also removed "Message" from the end of a contract name.

## References

* #0000

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto [hunk 1]
```diff
@@ -25,7 +25,7 @@ option csharp_namespace = "Energinet.DataHub.MeteringPoints.IntegrationEventCont
 /**
  * Represents the creation of a metering point.
  */
-message MeteringPointCreatedEventMessage {
+message MeteringPointCreated {
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto:28]: We should probably adhere to the style guide: https://developers.google.com/protocol-buffers/docs/style

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto [hunk 2]
```diff
@@ -25,7 +25,7 @@ option csharp_namespace = "Energinet.DataHub.MeteringPoints.IntegrationEventCont
...
   string Gsrn = 1; // Unique metering point identifier.
   string meteringPointType = 2; // Defines the type of metering point created.
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto:30]: As a minimum casing should be consistent.

**Review by cmgrd — CHANGES_REQUESTED** [review]

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto (hunk 2 above)
Comment (erduna) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto:30]: We actually shouldn't have Gsrn number in our outgoing contracts as it doesn't adhere to entso-e standards, so I will change that too. Later PR for other cases.

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto (hunk 1 above)
Comment (erduna) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto:28]: You mean snake case for fields?

Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto:28]: Yes

**Review by Korgath — APPROVED** [review]

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto (hunk 1 above)
Comment (erduna) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto:28]: I have just checked other domains, and they do not use snake case either: https://github.com/Energinet-DataHub/geh-post-office/blob/main/source/Contracts/v1/DataAvailable.proto

Should we still use snake case or just fix casing to be consistent?

**Review by Korgath — APPROVED** [review]

**Review by cmgrd — APPROVED** [review]

**Pull request merged**